test/studies/shootout/fasta/kbrady/fasta-qio.chpl

```diff
@@ -114,27 +72,62 @@ class Random {
   }
 }
 
-var lookup : [0..LOOKUP_SIZE-1] Freq;
-var random = new Random();
+// Sequences to be randomly generated (probability table)
+var IUB : [0..14] Freq;
+IUB[0]  = new Freq(a, 0.27);
+IUB[1]  = new Freq(c, 0.12);
+IUB[2]  = new Freq(g, 0.12);
+IUB[3]  = new Freq(t, 0.27);
+IUB[4]  = new Freq(B, 0.02);
+IUB[5]  = new Freq(D, 0.02);
+IUB[6]  = new Freq(H, 0.02);
+IUB[7]  = new Freq(K, 0.02);
+IUB[8]  = new Freq(M, 0.02);
+IUB[9]  = new Freq(N, 0.02);
+IUB[10] = new Freq(R, 0.02);
+IUB[11] = new Freq(S, 0.02);
+IUB[12] = new Freq(V, 0.02);
+IUB[13] = new Freq(W, 0.02);
+IUB[14] = new Freq(Y, 0.02);
+
+var HomoSapiens : [0..3] Freq;
+HomoSapiens[0] = new Freq(a, 0.3029549426680);
+HomoSapiens[1] = new Freq(c, 0.1979883004921);
+HomoSapiens[2] = new Freq(g, 0.1975473066391);
+HomoSapiens[3] = new Freq(t, 0.3015094502008);
+
+// Scan operation
+proc sumAndScale(a :[?D]) {
+  var p : real = 0;
+  for item in a {
+    p += item.p;
+    item.p = p * LOOKUP_SCALE;
+  }
+  a[a.numElements-1].p = LOOKUP_SCALE;
```

Comment:
This may be a style question to address in a group setting, but I think we typically just use ".size" instead of ".numElements"